bakery/src/scripts/gdocify_book.py

```diff
@@ -139,7 +139,7 @@ def fix_jpeg_colorspace(doc, out_dir):
                     # do nothing if we cannot open the image
                     print('Warning: Could not parse JPEG image with PIL: ' + str(img_filename))
         else:
-            raise Exception('Error: Resource file not existing: ' + str(img_filename))
+            print('Warning: Resource file not existing: ' + str(img_filename))
```

Comment:
Question more to check my understanding than anything: Is the reason we get here because the xpath query is too loose? Would it work if we selected just `img` with `resource` in there since we know this comes after `checksum`, or are there some corner cases this wouldn't catch?

Also, as a minor question: Does this conversion in place imply the checksum names are now incorrect (I don't think it matters in practice today, but perhaps worth a note / comment if so just in case it comes up later)?

Reply:
It seems the checksum check is more loose. It creates checksums if images are existing but does not stop pipeline execution.
So there are images referenced in col26488 & col12081 which do not have a checksum (because they are non existing) but they break the gdocify step because it's more restrictive than the checksum check.

Ultimatively it's a content error (!) but the gdocify step after the checksum step should not be more restrictive and should not stop the pipeline.

Reply:
Hmm...maybe my question wasn't clear (or well formed 😃  ). Asking differently, shouldn't it be the case that all image references to `../resources/<filename>` exist by this point and any broken references will not use that prefix? If so, could we just query for that explicitly?

Reply:
ah... you're right! Sorry I did not understand it first.
I could simplify the xpath by assuming all existing resources are starting with `../resources` 🧐  and it should all work with an exception. I think that's a good idea.